firewood/src/merkle/node.rs

```diff
@@ -375,8 +384,27 @@ impl Storable for Node {
             None
         };
 
+        let mut start_index = TRIE_HASH_LEN + 1;
+        let end_index = start_index + size_of::<u64>();
+        #[allow(clippy::indexing_slicing)]
+        let encoded_len = u64::from_le_bytes(
```

Comment:
These are the dangerous parts. If you change the type of the length, you have to make sure to update everywhere. Please use the aforementioned alias.